rustls/src/anchors.rs

```diff
@@ -124,7 +124,13 @@ impl RootCertStore {
     /// include ancient or syntactically invalid certificates.
     ///
     /// Returns the number of certificates added, and the number that were ignored.
-    pub fn add_parsable_certificates(&mut self, der_certs: &[impl AsRef<[u8]>]) -> (usize, usize) {
+    pub fn add_parsable_certificates<C>(
+        &mut self,
+        der_certs: impl IntoIterator<Item = C>,
```

Comment:
Style: I think I would specify `C: AsRef<[u8]>` in the type argument declaration to avoid the added verbosity of the where clause.

Reply:
Done.